src/Microsoft.ML.FastTree/GamModelParameters.cs

```diff
@@ -300,12 +300,25 @@ public double[] GetFeatureBinUpperBounds(int featureIndex)
             return featureBins;
         }
 
+        /// <summary>
+        /// Get all the bin upper bounds.
+        /// </summary>
+        public double[][] GetBinUpperBounds()
+        {
+            double[][] featureBins = new double[_binUpperBounds.Length][];
```

Comment:
>_binUpperBounds.Length [](start = 48, length = 22)

Why ctor uses _binUpperBounds.Length and loop featureBins.Length?  If they disagree this would crash. If they agree then always use the same...

Reply:
Done.